config/samples/secret-agent_v1alpha1_secretagentconfiguration.yaml

```
@@ -84,10 +88,10 @@ spec:
             cmd: importkeystore
             sourcePath: "platform-ca/ca"
           - name: ssl-keypair
-            cmd: importkeystore 
+            cmd: importkeystore
             sourcePath: "ds/ssl-key-pair"
           - name: master-keypair
-            cmd: importkeystore  
+            cmd: import
```

should this be `importcert`?
```suggestion
            cmd: importcert
```

---

this is `importkeystore`, I've been refining this spec on the forgeops repo and will do a bigger add back to this repo. 

It's importkeystore because the master-key-pair is used for encryption/decryption on DS data.

---

`import` is not defined in the types file :(